layouts/shortcodes/text_target.html

```diff
@@ -0,0 +1,13 @@
+<!-- 
+  Copyright (c) 2021 Eclipse Foundation, Inc.
+
+  This program and the accompanying materials are made available under the
+  terms of the Eclipse Public License v. 2.0 which is available at
+  http://www.eclipse.org/legal/epl-2.0.
+
+  Contributors:
+    Eric Poirier <eric.poirier@eclipse-foundation.org>
```

Comment:
Glad to see Eric is working while on vacation! :) He should write your name here, you did this work!